src/components/Article.jsx

```diff
@@ -1 +1,35 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => {
```

Comment:
You should split props into multiple lines (Checklist https://github.com/mate-academy/react_article/pull/2)